frontend/language/src/nb.json

```diff
@@ -1292,17 +1292,22 @@
   "settings_modal.about_tab_name_label": "Navn på Applikasjonen",
   "settings_modal.about_tab_repo_description": "Dette er en unik ID for applikasjonen både i URler og i APIer.",
   "settings_modal.about_tab_repo_label": "Navn på Repository",
-  "settings_modal.access_control_tab_checkbox_description": "Du må velge en eller flere aktører som tilgang til å starte applikasjonen.",
-  "settings_modal.access_control_tab_checkbox_legend": "Aktørkrav *",
-  "settings_modal.access_control_tab_heading": "Tilgangsstyring",
+  "settings_modal.access_control_tab_checkbox_description": "Her velger du hvem som eier og kan starte appen. “Alle” er valgt som standard. Du må minst velge én.",
+  "settings_modal.access_control_tab_checkbox_legend_label": "Hvilke type parter skal ha lov til å starte denne appen? ",
+  "settings_modal.access_control_tab_heading": "Oppstartskontroll for appen",
+  "settings_modal.access_control_tab_help_text_heading": "I oppstartskontrollen velger du hva som skal kontrolles når appen starter.",
+  "settings_modal.access_control_tab_option_access_control_docs_link": "<0 href=\"https://docs.altinn.studio/app/development/logic/instantiation\">I dokumentasjonen kan du lære hvordan du setter andre kontroller for oppstart.</0>",
+  "settings_modal.access_control_tab_option_access_control_docs_link_text": "Velg eller lag andre tilleggskontroller:",
+  "settings_modal.access_control_tab_option_all_type_partner": "Alle typer",
```

Comment:
I dont understand what this text key means. Should it be; 
`settings_modal.access_control_tab_option_all_types` only?

Reply:
fixed